### PR TITLE
Alter dataLayer push values for player state change

### PR DESCRIPTION
**Pull request by Guntrisoft**
- Feedback from the analytics team suggested we needed to
  push slightly different values

### CODE REVIEW

**Comment by benlovell** [comment]
We test for the presence of these quite consistently elsewhere. Is that something we should be doing here too?

**Comment by Guntrisoft** [comment]
@benlovell as agreed, testing the change is rather tricky, and perhaps not worth the time to get it working

@benbarnett have added a `return` when a value is matched

File: app/assets/javascripts/video-player.js.es6 [hunk 1]
```diff
@@ -67,9 +67,18 @@
           const originalOnPlayerStateChange = this.onPlayerStateChange;
 
           this.onPlayerStateChange = function(stateId) {
+            let stateValue = 'UNKNOWN';
+
+            $.each(YT.PlayerState, function(key, value) {
+              if (value === stateId) {
+                stateValue = key;
+                return;
```
Comment (benbarnett) [inline app/assets/javascripts/video-player.js.es6]: Tiny thing but I think we need to actually `return false;` out of this: http://stackoverflow.com/questions/1799284/how-to-break-exit-from-a-each-function-in-jquery

Comment (Guntrisoft) [inline app/assets/javascripts/video-player.js.es6]: sorted

**Pull request merged**